src/components/Home.tsx

```diff
@@ -1,7 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Do not disable `eslint` errors like this @subhamtsd

Reply:
Ressolved the same